### PR TITLE
csi-driver presubmits

### PR DESCRIPTION
**Pull request by rgolangh**
Add presubmits that invokes tests, functional tests, and goveralls.

### CODE REVIEW

File: github/ci/prow/files/jobs/kubevirt/csi-driver/csi-driver-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+presubmits:
+  kubevirt/csi-driver:
+    - name: pull-kubevirt-e2e-k8s-1.20
```
Comment (dhiller) [inline github/ci/prow/files/jobs/kubevirt/csi-driver/csi-driver-presubmits.yaml:3]:
```suggestion
    - name: pull-csi-driver-e2e-k8s-1.20
```

File: github/ci/prow/files/jobs/kubevirt/csi-driver/csi-driver-presubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+              - "/bin/sh"
+              - "-c"
+              - "export TARGET=k8s-1.20 && make test-functional"
+            # docker-in-docker needs privileged mode
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
+    - name: pull-kubevirt-goveralls
```
Comment (dhiller) [inline github/ci/prow/files/jobs/kubevirt/csi-driver/csi-driver-presubmits.yaml:37]:
```suggestion
    - name: pull-csi-driver-goveralls
```

File: github/ci/prow/files/jobs/kubevirt/csi-driver/csi-driver-presubmits.yaml (hunk 1 above)
Comment (rgolangh) [reply github/ci/prow/files/jobs/kubevirt/csi-driver/csi-driver-presubmits.yaml]: done

**Comment by rgolangh** [comment]
@dhiller I wasn't able to rehearse or locally invoke it with phaino. Please approve or lgtm this so I can at least work with the presubmit invocation on the csi-driver repo.

**Comment by dhiller** [comment]
/lgtm
/approve

Let's see then :)

**Pull request merged**